### PR TITLE
refactor: remove ChainId optional

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/hooks/index.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export function useActiveWeb3React(): {
         : undefined,
     [isConnectedSolana, connectedWalletSolana?.adapter],
   )
-  const mockAccountEVM = ''
+  const mockAccountEVM = '0x8ee4c695f64c5f0e549634a9393656f8a6f5efc6'
```
Comment (nguyenhoaidanh) [inline src/hooks/index.ts:85]: rm

Comment (namgold) [reply src/hooks/index.ts]: dinh vai

File: src/pages/Farm/index.tsx [hunk 2]
```diff
@@ -189,7 +189,7 @@ const Farm = () => {
   const token0 = useCurrency(token0Id)
   const token1 = useCurrency(token1Id)
 
-  if (!isEVM) return <Navigate to="/" />
+  if (!isEVM) return <Navigate to="/swap" />
```
Comment (nguyenhoaidanh) [inline src/pages/Farm/index.tsx:192]: use APP_PATHS and add network route

Comment (namgold) [reply src/pages/Farm/index.tsx]: reverted

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**